### PR TITLE
Refactor the reclaim_policy_manager and fix TiKVGroup.GetName()

### PR DESCRIPTION
**Pull request by lonng**
Signed-off-by: Lonng <heng@lonng.org>

### What problem does this PR solve? 

The previous implementation contains a BUG that wrongly treats `TiKVGroup.GetName()` as `namespace`.

### What is changed and how does it work?

This PR fixes the bug described above and refactor some code to simply the codebase.

### Check List 

Tests 

 - Unit test
 - E2E test
 - Stability test

Related changes

 - Need to cherry-pick to the release branch

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: pkg/manager/meta/reclaim_policy_manager.go [hunk 1]
```diff
@@ -15,6 +15,8 @@ package meta
 
 import (
 	"fmt"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
```
Comment (Yisaer) [inline pkg/manager/meta/reclaim_policy_manager.go:19]: plz fmt the goimports.

**Comment by lonng** [comment]
@Yisaer Do we need to cherry-pick this PR?

**Comment by Yisaer** [comment]
@lonng  I think we can. As `TiKVGroup` is only used in master branch now, I think you need resolve conflict if you cherry-pick it into 1.1 version.

**Comment by lonng** [comment]
@Yisaer OK

**Pull request merged**